nextflow.config

```diff
@@ -248,7 +253,7 @@ manifest {
     description     = """Workflow converts one or multiple bam/cram files to fastq format"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version         = '2.1.1'
+    version         = '2.1.2dev'
```

Comment:
why 2.1.1 to 2.1.2dev?

Reply:
I just say that you did something similar in the "back to dev"-PR's for Sarek. Anything wrong there?

Reply:
yeah, but that's the TEMPLATE sync PR, not the back to dev PR

Reply:
If it bothers you that it is there, then I'll revert it.

Reply:
no, I'm fine no worries, but then you might want to do an actual `nf-core bump-version 2.1.2dev`